bog_app/app/views/creatures/index.html.erb

```diff
@@ -0,0 +1,30 @@
+
+
+<%= form_tag(creatures_path, :method => "get", id: "search-form") do %>
+<%= text_field_tag :search, params[:search], placeholder: "Search Posts" %>
```

Comment:
The contents of your form tag (the text field and submit) should be indented one place.